location_extractor/clients.py

```diff
@@ -1,12 +1,18 @@
+import csv
 import os
 import sqlite3
+
 from dataclasses import dataclass
-from sqlite3 import Cursor, Connection
-from typing import List, Tuple, Union, Optional
+from typing import Iterable, List, Optional, Tuple, Union
+
+from typing_extensions import Final
 
 from location_extractor import src_dir
-from location_extractor.helpers import remove_accents
-import csv
+from location_extractor.utils import remove_accents
+
+StringOrIterableOfStrings = Union[str, Iterable[str]]
+COMMA: Final = ','
```

Comment:
just to avoid violation [`WPS226`](https://wemake-python-stylegui.de/en/latest/pages/usage/violations/complexity.html#wemake_python_styleguide.violations.complexity.OverusedStringViolation)